build-tests/rush-stack-compiler-3.7-library-test/package.json

```diff
@@ -11,7 +11,6 @@
   "devDependencies": {
     "@microsoft/node-library-build": "workspace:*",
     "@microsoft/rush-stack-compiler-3.7": "workspace:*",
-    "@types/node": "10.17.13",
-    "gulp": "~4.0.2"
```

Comment:
Gulp is still used by the `"build"` command above -- how are we able to remove it?

Reply:
Oops.